src/core/utils/helpers/list-mapper.ts

```diff
@@ -108,10 +110,14 @@ export const mapProductToBasicDetailsType = (
     externalProductId
   } = material;
 
+  const {
```

Comment:
Sucks that you need to reintroduce this. But I understand that things need to move along.
I think you should put a TODO here about getting rid of it again in the future and the reason why it must be solved like this for now.